### ISSUE TITLE
Typescript definitions

### ISSUE THREAD

**Issue by schoolboyqueue** [LGSInnovations/react-sigplot#13]

**Comment by mrecachinas** [comment]
Hi @schoolboyqueue, just to make sure, you're asking for Typescript definitions to be added to React-SigPlot?

I will try to get around to that. One issue is this might require SigPlot.js to have Typescript definitions too.

**Comment by schoolboyqueue** [comment]
Yes, I believe it would be greatly beneficial to have typescript definitions for SigPlot.js as a baseline which would make it easier to have them for this project. 

Side note: There exists a GitHub project that has attempted to do this already but their definitions are out of date. https://github.com/geontech/sigplot-ts

I use this library in projects at work (because its awesome) and our entire baseline is written in typescript.

**Comment by maihde** [comment]
I’m supportive of moving Typescript definitions into the main code base so
they can stay updated.

Michael

**Comment by mrecachinas** [comment]
@schoolboyqueue Sounds good. I’ll start to work on this tomorrow.

@maihde We should discuss the SigPlot.js ts definitions.
